app/models/build_part.rb

```diff
@@ -96,6 +96,10 @@ def elapsed_time
     end
   end
 
+  def as_json(options={})
+    super(options.merge(methods: :status))
```

Comment:
can you change this to a `reverse_merge`? As written its impossible to pass in options with `methods` specified.